packages/devtools/lib/src/ui/theme.dart

```diff
@@ -40,6 +40,8 @@ class ThemedColor implements Color {
 
   Color get _current => isDarkTheme ? _dark : _light;
 
+  ThemedColor reverse() => ThemedColor(_dark, _light);
```

Comment:
It is nit picky, but I think we shouldn't add a method like this.
Unfortunately, light and dark themes aren't as simple as reversing colors in general.
Having this method will encourage the wrong patterns.

Reply:
removed